packages/accounts/src/light-account/account.ts

```diff
@@ -118,7 +119,9 @@ export async function createLightAccount({
     });
 
     if (storage == null) {
-      throw new Error("could not get storage");
+      throw new FailedToGetStorageSlotError(
+        "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc"
```

Comment:
can we add a link as a comment to reference for this address?

Reply:
yea good call, updating now